src/pages/api/tour.ts

```diff
@@ -0,0 +1,68 @@
+import { Prisma, Tour } from "@prisma/client";
+import type { NextApiRequest, NextApiResponse } from "next";
+import { getToken } from "next-auth/jwt";
+import { prisma } from '../../lib/prisma'
+
+// API Inputs.
+export interface GetTourRequestType {
+  firstName: string;
+  lastName: string;
+  email: string;
+}
+
+// API Outputs.
+export interface GetTourResponseType {
+  tours: Tour[];
+}
+
+export interface CreateTourRequestType {
+  userId: string;
+}
+
+export interface CreateTourResponseType {
+  tourId?: string;
+  error?: string;
+}
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse<GetTourResponseType | CreateTourResponseType>
+) {
+  const session = await getToken({ req });
+  //if (!session) return res.status(405).json({ error: "Not authorized." });
```

Comment:
delete if not necessary anymore

Reply:
Gian has to fix these endpoints with the getToken attributes.